Source/WebCore/PAL/pal/spi/cocoa/AVKitSPI.h

```diff
@@ -387,3 +387,32 @@ NS_ASSUME_NONNULL_BEGIN
 NS_ASSUME_NONNULL_END
 #endif // USE(APPLE_INTERNAL_SDK)
 #endif // ENABLE(WIRELESS_PLAYBACK_TARGET) && HAVE(AVROUTEPICKERVIEW)
+
+// AVPictureInPicture SPI
+#if USE(APPLE_INTERNAL_SDK)
+
+#if PLATFORM(IOS_FAMILY)
+#import <AVKit/AVPictureInPictureController_GenericSupport.h>
+#endif
+
+#else
+
+#if PLATFORM(IOS_FAMILY)
+NS_ASSUME_NONNULL_BEGIN
+
+@interface AVPictureInPictureContentViewController : UIViewController
+@property (nonatomic, strong, readonly, nullable) AVPlayerController *playerController;
+@end
+
+@interface AVPictureInPictureControllerContentSource (GenericSupport)
+- (instancetype)initWithSourceView:(UIView *)sourceView contentViewController:(AVPictureInPictureContentViewController *)contentViewController playerController:(__kindof AVPlayerController *)playerController API_AVAILABLE(ios(16.0),tvos(16.0)) API_UNAVAILABLE(macos, watchos);
+@property (nonatomic, weak, readonly, nullable) UIView * activeSourceView API_AVAILABLE(ios(16.0),tvos(16.0)) API_UNAVAILABLE(macos, watchos);
+@property (nonatomic, readonly, nullable) __kindof AVPictureInPictureContentViewController * activeContentViewController API_AVAILABLE(ios(16.0),tvos(16.0)) API_UNAVAILABLE(macos, watchos);
```

Comment:
Nit: Don't we remove the space after the `*`?

Reply:
done